allure-generator/src/main/javascript/plugins/tab-timeline/TimelineView.js

```diff
@@ -80,13 +81,16 @@ class TimelineView extends BaseChartView {
                 })
             );
 
+        const opts = {
+            count: this.data.selectedTestCases,
+            percent: (100 * this.data.selectedTestCases / this.model.get('statistic').total).toFixed(2),
+            duration: duration(this.selectedDuration)
+        };
         this.slider.insert('g')
             .append('text')
             .attr('transform', `translate(${this.width/2}, 20)`)
             .attr('class', 'timeline__slider_text')
-            .text(`Selected ${this.data.selectedTestCases}
-                (${(100*this.data.selectedTestCases/this.model.get('statistic').total).toFixed(2)}%)
-                tests with duration above ${duration(this.selectedDuration)}`);
+            .text(t('tab.timeline.selected', {hash: opts}));
```

Comment:
Maybe it is better to call `i18next` directly. Then you will not need to wrap this into hash.

Don't know what is better, just commenting.